app/script/conversation/ConversationCellState.js

```diff
@@ -323,7 +323,7 @@ z.conversation.ConversationCellState = (() => {
       }
     },
     icon: () => z.conversation.ConversationStatusIcon.UNREAD_MESSAGES,
-    match: conversationEntity => conversationEntity.unreadState().allEvents.length > 0,
+    match: conversationEntity => conversationEntity.unreadState().allMessages.length > 0,
```

Comment:
This generator never consumes the `allEvents` array (only the `allMessages`). So I think it's semantically more correct to use it as a condition to start the computation.

I guess, right now, we can have a scenario (where we only have system events for example), where we go through the computation loop for nothing.